configs/settings/dev.py

```diff
@@ -0,0 +1,19 @@
+from .base import *
+import environ
+
+environ.Env.read_env(os.path.join(BASE_DIR, 'env', '.env.dev'))
+
+DATABASES = {
+    "default": env.db(
+        "DATABASE_URL",
+        default=f"sqlite:///{BASE_DIR / 'db.sqlite3'}"
+    )
+}
```

Comment:
❗ django-environ 문법이 잘못되었습니다. 공식문서를 다시 읽어주세요. https://django-environ.readthedocs.io/en/latest/quickstart.html

```suggestion
DATABASES = {
    "default": env.db()
}
```
이렇게 작성하거나
```suggestion
DATABASES = {
    "default": env.db_url(
        "DATABASE_URL",
        default=f"sqlite:///{BASE_DIR / 'db.sqlite3'}"
    )
}
```
이렇게 작성해야 합니다.